### ISSUE TITLE
Support unwrapping error

### ISSUE THREAD

**Issue by jaycetde** [bugsnag/bugsnag-go#193]
### Description

Support unwrapping errors using `errors.Unwrap` from standard library.  Supporting unwrap will also allow `errors.Is` to work:
```go
import (
	"errors"
	bserrors "github.com/bugsnag/bugsnag-go/v2/errors"
)

var MyErr = errors.New("my error")

wrappedErr := bserrors.Errorf("wrapped: %w", MyErr)

if errors.Is(wrappedErr, MyErr) {
  // Custom logic here
}
```

**Describe the solution you'd like**
Add the following to errors/error.go:
```go
func (err *Error) Unwrap() error {
	return err.Err
}
```
or possibly:
```go
func (err *Error) Unwrap() error {
	return errors.Unwrap(err.Err)
}
```

**Comment by luke-belton** [comment]
Hi @jaycetde - thanks for raising this. We've added an item to our product roadmap to work on this in the future. I don't currently have an ETA, but we'll keep this thread updated 👍

**Comment by luke-belton** [comment]
Hi @jaycetde - just wanted to follow up on this to say we're unlikely to prioritise this work in the immediate future, but we would be willing to accept a contribution if you're interested. We'd want to go for your second proposed solution, i.e.
```
func (err *Error) Unwrap() error {
	return errors.Unwrap(err.Err)
}
```
The solution would also need to have a simple test to cover it 👍

**Comment by luke-belton** [comment]
Hi @jaycetde - thanks again for this, [support for this](https://github.com/bugsnag/bugsnag-go/pull/194) was released in [`v1.9.1`](https://github.com/bugsnag/bugsnag-go/releases/tag/v1.9.1) of `bugsnag-go` 👍
